### ISSUE TITLE
Heading in rtf. 

### ISSUE THREAD

**Issue by deep125** [doxygen/doxygen#6522]
rtf output incorrectly displays the heading.  Wrong styles are set in the RTF document
[doxygen_bag_2.zip](https://github.com/doxygen/doxygen/files/2392948/doxygen_bag_2.zip)
example and project and result attached

**Comment by albert-github** [comment]
A bit more description on what is, in your opinion, wrong wouldn't hurt.
As far as I can see the problem might be the size of the headings in the RTF document ?

**Comment by deep125** [comment]
This fix helped me
```
 src/rtfdocvisitor.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/rtfdocvisitor.cpp b/src/rtfdocvisitor.cpp
index 7fbfdc8d..0be22666 100644
--- a/src/rtfdocvisitor.cpp
+++ b/src/rtfdocvisitor.cpp
@@ -1072,7 +1072,7 @@ void RTFDocVisitor::visitPre(DocHtmlHeader *header)
   m_t << "{" // start section
       << rtf_Style_Reset;
   QCString heading;
-  int level = QMIN(header->level()+2,4);
+  int level = QMIN(header->level(),5);
   heading.sprintf("Heading%d",level);
   // set style
   m_t << rtf_Style[heading]->reference;
```

**Comment by deep125** [comment]
Need info?

**Comment by albert-github** [comment]
The question in my earlier comment has not been answered, that is why I asked for more information.
Please also compare the implementation in the same file in the routine  `void RTFDocVisitor::visitPre(DocSection *s)` the level.

Can you make a pull request out of your patch?

**Comment by deep125** [comment]
I answered and sent you a patch code. I can't do pull request, sorry.

**Comment by albert-github** [comment]
I can see that you edited the original issue (already on September 18, 2018) but this is, unfortunately, badly signaled by github and is easily overlooked (as I did). The patch I did see.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.15.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
